packages/fast-components-react-base/src/radio/radio.props.ts

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import { IManagedClasses, IRadioClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface IRadioHandledProps {
+
+    /**
+     * Unique Id
+     */
+    id?: string;
```

Comment:
I think the ID should be required for accessability

Reply:
changed to be required